Assets/Scripts/Models/Buildable/BuildableJobs.cs

```diff
@@ -154,6 +154,11 @@ public bool HasJobWithPredicate(Func<Job, bool> predicate, out Job job)
         return job != null;
     }
 
+    public bool WorkspotIsInternal()
```

Comment:
Can we name this `WorkSpotIsInternal`? Or `IsWorkSpotInternal` better? (uppercase S for Spot)
Does it needs docs?